libcodechecker/analyze/log_parser.py

```diff
@@ -170,8 +170,14 @@ def get_compiler_includes(parseLogOptions, compiler, lang, compile_opts,
     # The first sysroot flag found among the compilation options is added
     # to the command below to give a more precise default include list.
     # Absence of any sysroot flags results in an empty string.
-    sysroot = next(
-        (item for item in compile_opts if item.startswith("--sysroot=")), "")
+    sysroot = ""
```

Comment:
Shouldn't we fix it in the https://github.com/Ericsson/codechecker/blob/master/libcodechecker/log/option_parser.py
instead? There is a code block which already handles the sysroot parameters and merging of them.